cmd/statusd/main.go

```diff
@@ -73,13 +74,30 @@ var (
 // All general log messages in this package should be routed through this logger.
 var logger = log.New("package", "status-go/cmd/statusd")
 
-func levelFromString(level string) log.Lvl {
-	lvl, err := log.LvlFromString(strings.ToLower(level))
+func enhanceLogger(logger *log.Logger, config *params.NodeConfig) error {
+	var (
+		handler log.Handler
+		err     error
+	)
+
+	if config.LogFile != "" {
+		handler, err = log.FileHandler(config.LogFile, log.LogfmtFormat())
+		if err != nil {
+			return err
+		}
+	} else {
+		handler = log.StreamHandler(os.Stdout, log.TerminalFormat(true))
```

Comment:
are you sure that it is common to stream logs to stdout actually? in my experience, it is usually stderr

Reply:
from http://pubs.opengroup.org/onlinepubs/9699919799/functions/stderr.html

> standard output (for writing conventional output), and standard error (for writing diagnostic output)

but i think it is only important for command line applications, so whatever you choose

Reply:
`log` package functions from the standard lib use stderr https://golang.org/src/log/log.go?s=5964:6020#L73.

Reply:
yes `log.Print...` would write to stderr, I'll change it